src/tests/diagnosticUtil.ts

```diff
@@ -24,48 +24,25 @@ export function formatTestErrors(error: Error): Error {
   return error;
 }
 
-export function getSyncDiagnostic(syncRecord: SyncTestFailure): ApexDiagnostic {
-  const diagnostic: ApexDiagnostic = {
-    exceptionMessage: syncRecord.message,
-    exceptionStackTrace: syncRecord.stackTrace,
-    className: syncRecord.stackTrace
-      ? syncRecord.stackTrace.split('.')[1]
-      : undefined,
-    compileProblem: ''
-  };
+export function getDiagnostic(
```

Comment:
the 2 objects are separated only by some capitalization differences.  Made sense to combine the fn